core/platform/auth/firebase_auth_services.py

```
@@ -648,3 +745,22 @@ def _create_auth_claims(
     )
+
+
+def _log_delete_users_result(result: firebase_auth.DeleteUsersResult) -> None:
+    """Logs the results of a delete_users() call.
+
+    Args:
+        result: firebase_auth.DeleteUsersResult. The result of the deletion.
+    """
+    logging.info(
+        '[FIREBASE] Succesfully deleted %s users', result.success_count
+    )
```

Ruff (0.14.14)

[warning] `info()` call on root logger

(LOG015)